setup.cfg

```diff
@@ -7,7 +7,7 @@ passenv = *
 deps = -r{toxinidir}/requirements.txt
        -r{toxinidir}/tests/requirements.txt
 commands = python3 -B -m pytest {tty:--color=yes} {posargs} -v --mypy \
-           --cov=aiojenkins --cov=tests --cov-report=term --ignore docs
+           --cov=aiojenkins --asyncio-mode=auto --cov=tests --cov-report=term --ignore docs
```

Comment:
Removes the need to include `@pytest.mark.asyncio` everywhere

Reply:
Looks good